src/mainviews/create-job.tsx

```diff
@@ -576,10 +577,15 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
       jobOptions.parameters = serializeParameters(props.model.parameters);
     }
 
-    api.createJob(jobOptions).then(response => {
-      // Switch to the list view with "Job List" active
-      props.showListView('Job');
-    });
+    api
+      .createJob(jobOptions)
+      .then(response => {
+        // Switch to the list view with "Job List" active
+        props.showListView('Job');
+      })
+      .catch((error: string) => {
+        props.handleModelChange({ ...props.model, createError: error });
+      });
   };
```

Comment:
Believe we want to briefly immediately `createError` upon form submission. This is consistent with the UX in most other forms I've used. If the user gets a create error, and then retries to submit the form, this behavior provides additional visual emphasis on the fact that another create error happened, rather than having the user think that the form's buggy. To do this, just append this snippet below line 589:

```
props.handleModelChange(model => ({ ...model, createError: undefined }));
```